src/main/java/frc/robot/subsystems/swerve/modules/ModuleUtils.java

```diff
@@ -57,9 +53,14 @@ public static double velocityToOpenLoopVoltage(
      * @param moduleAngle the angle or velocity in angle of the module
      * @return the distance or velocity without the coupling
      */
-    public static double removeCouplingFromRevolutions(Rotation2d drivePosition, Rotation2d moduleAngle, double couplingRatio) {
+    public static Rotation2d removeCouplingFromAngle(Rotation2d drivePosition, Rotation2d moduleAngle, double couplingRatio) {
```

Comment:
this function's name makes it sound lika a void function.
consider changing the name to `getUncoupledAngle` or something like that